### ISSUE TITLE
Table row and cell contextual class priority

### ISSUE THREAD

**Issue by svallis** [twbs/bootstrap#15986]
When specifying one of the contextual classes (.info, .success, .warning, .danger), I would expect the class specified at the cell level to override the one set at the row level. This is the case for .danger, but not the case for the others at present. In the example below you can see that in row two and three column three overrides the cell styles with the row style, whereas column four exhibits the behaviour I would expect to see:

http://www.bootply.com/2uFui3H1ta

**Comment by twbs-lmvtfy** [comment]
Hi @svallis!

You appear to have posted a live example (http://s.bootply.com/render/2uFui3H1ta), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 9, column 9: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.2 ; saw what appears to be usage of Bootstrap 3.3.1
* line 73, column 9: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.2 ; saw what appears to be usage of Bootstrap 3.3.1

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by svallis** [comment]
Updated live example:

http://www.bootply.com/SJ7S3DkPuU

**Comment by cvrebert** [comment]
Duplicate of #12418. Nested usage of the table contextual background color classes is not supported.
